releasetasks/templates/bb_update_verify.yml.tmpl

```diff
@@ -27,6 +27,10 @@
                 build_number: {{ buildNumber }}
                 revision: "{{ revision }}"
                 release_promotion: true
+                script_repo_revision: "{{ "{product}_{version}_BUILD{buildNumber}_RUNTIME".format(
+                    product=product.upper(),
+                    version=version,
```

Comment:
You need to replace dots with underscores in version, something like `version.replace(".", "_")`